src/lib.rs

```diff
@@ -26,6 +28,9 @@ pub fn _eq(a: u64, b: u64) -> bool {
     (a as i64).wrapping_sub(b as i64).abs() <= 1
 }
 
+#[inline(never)]
+fn force_eval<T>(_: T) {}
```

Comment:
I believe the compiler is still free to opt away a function call to this function because it has no side effects and doesn't modify the input. I think something closer to the MUSL macro would be something like this:

``` rust
#[inline(always)]
fn force_eval<T>(x: &T) {
    unsafe { ptr::read_volatile(x); }
}
```

And you would invoke it like this: `force_eval(&(x + TOINT));`